.github/workflows/pr-percy-prepare-label.yml

```
@@ -0,0 +1,22 @@
+# This workflow ensures Percy is executed against PRs with the Percy required label, regardless of which paths were changed
+name: "Percy [labeled] / Prepare build"
```

That's weird, I thought I commented on this already.

I meant for the workflow to be `Percy [labeled]`, and for the job to be `Prepare build`, as now we still have a long and unnecessarily detailed "Copy changed files to GHA artifact" as status name on PR.

```suggestion
name: "Percy [labeled]"
```

---

Ah I misunderstood your original comment and named the entire workflow that; this has been revised!